sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -661,6 +661,23 @@ public DigitalTwinsServiceVersion getServiceVersion() {
             });
     }
 
+    /**
+     * Gets all the relationships referencing a digital twin as a target by iterating through a collection.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.digitaltwins.core.asyncClient.listIncomingRelationships#String}
```

Comment:
Is the plan to add readme later which refers to these snippets?

Reply:
No, just to have these javadocs point to the snippets.